compose-examples/prysm-only/docker-compose.yaml

```diff
@@ -100,15 +100,14 @@ services:
     environment:
       GF_RENDERING_SERVER_URL: http://renderer:8082/render
       GF_RENDERING_CALLBACK_URL: http://grafana:3000
-      GF_LOG_FILTERS: rendering:debug
     <<: *logging
 
   renderer:
-    image: grafana/grafana-image-renderer:2.0.0
+    image: grafana/grafana-image-renderer:${GRAFANA_IMAGE_RENDERER_TAG}
     restart: on-failure
     hostname: renderer
     ports:
-      - 127.0.0.1:8082:8082
+      - 8082
```

Comment:
I think this is not necessary at all... is there any service accessing this container from outside docker?

Reply:
at the moment only "grafana" service accessing it

Reply:
but grafana is inside docker, not from outside of the host. Can you please try remove this port binding and see if it still works?

Reply:
yeah sure, just minut

Reply:
it's removed and works fine